src/styles/lineStyle.js

```diff
@@ -11,6 +95,38 @@ function lineStyle(linesymbolizer) {
   let style = {};
   if (linesymbolizer.stroke) {
     style = linesymbolizer.stroke.styling;
+
+    if (linesymbolizer.stroke.graphicstroke) {
+      return new Style({
+        renderer: (pixelCoords, renderState) => {
+          // TODO: Error handling, alternatives, etc.
+          const render = toContext(renderState.context);
+
+          const pointStyle = getPointStyle(linesymbolizer.stroke.graphicstroke, renderState.feature);
+
+          const size = expressionOrDefault(linesymbolizer.stroke.graphicstroke.graphic.size, DEFAULT_POINT_SIZE); // TODO: Dynamic size?
+          let multiplier = 1; // default, i.e. a segment is the size of the graphic (without stroke/outline).
+
+          // Use strokeDasharray to space graphics. First digit represents size of graphic, second the relative space, e.g.
+          // size = 20, dash = [2 6] -> 2 ~ 20 then 6 ~ 60, total segment length should be 20 + 60 = 80
+          if (linesymbolizer.stroke.styling && linesymbolizer.stroke.styling.strokeDasharray) {
+            const dash = linesymbolizer.stroke.styling.strokeDasharray.split(' ');
+            if (dash.length >= 2 && dash[0] !== 0) {
+              multiplier = dash[1] / dash[0] + 1;
+            }
+          }
+
+          const splitPoints = splitLineString(new LineString(pixelCoords), multiplier * size,
+            { alwaysUp: true, midPoints: false, extent: render.extent_ }); // TODO: Check .getExtent()
```

Comment:
Please add `// eslint-disable-next-line no-underscore-dangle` to make ESLint happy.